content/en/news/releases/1.8.x/announcing-1.8/upgrade-notes/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.8.
+weight: 20
+---
+
+When you upgrade from Istio 1.7.x to Istio 1.8.x, you need to consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.7.x.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.7.x.
+
+## Mixer is no longer supported in Istio
+
+If you are using the `istio-policy` or `istio-telemetry` services, or any
+related Mixer configuration, you will not be able to upgrade without taking
+action to either (a) convert your existing configuration and code to the new
+extension model for Istio or (b) using the gRPC shim developed to bridge
+transition to the new model. For more details, please refer the docs on
+istio.io.
```

Comment:
Can we add a link to these docs?

Reply:
We need to change this to point to: https://github.com/istio/istio/wiki/Enabling-Envoy-Authorization-Service-and-gRPC-Access-Log-Service-With-Mixer